packages/http/src/models/Hotspot.ts

```diff
@@ -25,7 +25,12 @@ export interface HTTPHotspotObject {
   block_added?: number
   geocode?: HTTPGeocodeObject
   address: string
-  status?: Status
```

Comment:
should there be a different snake-cased-fields interface called HTTPStatusObject to match the way HTTPGeocodeObject is done? not a huge deal either way, and just trying to understand the code better (still getting up to speed with TypeScript and helium-js, so this comment is more for my own understanding)

Reply:
Or should we just update the type that is already there for Status?

Reply:
The Status that was there is the camelCased status. For the HTTP object we need snake case. We could create an interface `HTTPStatusObject`, but seeing as though it is only used in this context I didn't. If we ever wanted to reuse it, then we definitely would want to create a separate interface.

Reply:
Ah its only for the HTTP type, ok this is fine. Should we update that status object in the hotspot to include the new camel case parameters? I think its missing `listenAddrs` ?

Reply:
Wait nevermind its there lol